src/Admin/Views/Providers/AddExistingOrganization.cshtml

```diff
@@ -0,0 +1,97 @@
+@using Bit.SharedWeb.Utilities
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model OrganizationSearchViewModel
+
+@{
+    ViewData["Title"] = "Add Existing Organization";
+    var providerId = ViewContext.RouteData.Values["id"];
+}
+
+<h1>Add Existing Organization</h1>
+<div class="row mb-2">
+    <div class="col">
+        <form class="form-inline mb-2" method="get" asp-route-id="@providerId">
+            <label class="sr-only" asp-for="OrganizationName">@Html.DisplayNameFor(m => m.OrganizationName)</label>
+            <input type="text" class="form-control mb-2 mr-2 flex-fill" placeholder="@Html.DisplayNameFor(m => m.OrganizationName)" asp-for="OrganizationName" name="name">
+            <label class="sr-only" asp-for="OrganizationOwnerEmail">@Html.DisplayNameFor(m => m.OrganizationOwnerEmail)</label>
+            <input type="email" class="form-control mb-2 mr-2 flex-fill" placeholder="@Html.DisplayNameFor(m => m.OrganizationOwnerEmail)" asp-for="OrganizationOwnerEmail" name="ownerEmail">
+            <button type="submit" class="btn btn-primary mb-2" title="Search" formmethod="get"><i class="fa fa-search"></i> Search</button>
+        </form>
+    </div>
+</div>
+
+<form method="post" id="select-form" asp-route-id="@providerId">
+    <div class="table-responsive">
+        <table class="table table-striped table-hover">
+            <thead>
+            <tr>
+                <th style="width: 20px;">All</th>
+                <th>Name</th>
+                <th style="width: 190px;">Plan</th>
+            </tr>
+            </thead>
+            <tbody>
+            @if (!Model.Items.Any())
+            {
+                <tr>
+                    <td colspan="5">No results to list.</td>
+                </tr>
+            }
+            else
+            {
+                @for (var i = 0; i < Model.Items.Count; i++)
+                {
+                    <tr>
+                        <td class="text-center">
+                            @Html.HiddenFor(m => Model.Items[i].Id)
```

Comment:
I don't see any way around this one, but please make it `readonly` and `autocomplete="off"` as an extra precaution